### ISSUE TITLE
Unable to re-publish

### ISSUE THREAD

**Issue by g964** [Codewars/codewars.com#558]
I got lots of

```
Publishing Timed Out!
The server timed out while trying to publish your changes.
CANCEL
```

and it is impossible to re-publish.

**Comment by DonaldKellett** [comment]
Same (#559).  I just authored a new kata in PHP and was about to publish it but I tried at least 10 times and it just kept reporting a Timeout error.

**Comment by jhoffner** [comment]
This is an reoccurring issue with background processing failures, where the process locks up. Its a very tricky thing to debug for since I can't reproduce (just days and days of production scale traffic causes this issue eventually). I will continue to keep monitoring and debugging.

**Comment by jhoffner** [comment]
Related to #544

**Comment by tachyonlabs** [comment]
I was just typing a long description about how after adding Java to my new kata, every time I would click Re-Publish, after "Please wait ..." would spin for a bit I would get the timeout. But then when I finished writing, and thought I would try Re-Publish "just one more time" before saving my comment, it finally worked! :-)

**Comment by brendanvos** [comment]
@g964 @jhoffner @kazk Can likely be closed as duplicate #544 was closed by jhoffner.

**Comment by kazk** [comment]
Closing this because this is a duplicate of #544 and it has been fixed.
